frontend/src/components/session/session_form.css

```diff
@@ -0,0 +1,155 @@
+.session-form-content-container {
```

Comment:
one thing I wanted to note: their SessionForm seems to have some loading effects (inputs go gray, button goes pale)